bpf/lib/policy.h

```diff
@@ -285,8 +292,10 @@ __policy_can_access(const void *map, struct __ctx_buff *ctx, __u32 local_id,
 	/* L3 lookup: an exact match on L3 identity and LPM match on L4 proto and port. */
 	policy = map_lookup_elem(map, &key);
 
-	/* L3 policy can be chosen without the 2nd lookup if it is a deny. */
-	if (likely(policy && policy->deny)) {
+	/* L3 policy can be chosen without the 2nd lookup if it has the highest possible precedence
+	 * value (which implies that it is a deny).
```

Comment:
In a previous version of this patchset, I used precedence = 0 as a signal that precedence was disabled. Is that still something we want to do?

Reply:
Yes, will do.

Reply:
Done.